build.gradle

```diff
@@ -1,22 +1,22 @@
 buildscript {
     ext {
-        kotlin_version = '1.8.22'
+        kotlin_version = '1.9.23'
         coroutinesVersion = '1.6.4'
         ok_http_version = '4.9.1'
         dashjVersion = '20.0.4'
-        hiltVersion = '2.45'
+        hiltVersion = '2.51'
+        hiltCompilerVersion = '1.2.0'
         hiltWorkVersion = '1.0.0'
         workRuntimeVersion='2.7.1'
         firebaseVersion = '32.1.1'
-        roomVersion = '2.4.3'
+        roomVersion = '2.6.1'
```

Comment:
This was changed in hopes it might eliminate this:
``` 
e: [ksp] [MissingType]: Element 'org.dash.wallet.common.data.RoomConverters' references a type that is not present
e: [ksp] androidx.room.RoomKspProcessor was unable to process 'org.dash.wallet.features.exploredash.ExploreDatabase' because not all of its dependencies could be resolved. Check for compilation errors or a circular dependency with generated code.
e: Error occurred in KSP, check log for detail
```

Similar error: https://issuetracker.google.com/issues/274889425